pkg/plugins/plugins.go

```diff
@@ -373,17 +373,7 @@ func (scanner *PluginScanner) IsBackendOnlyPlugin(pluginType string) bool {
 
 // validateSignature validates a plugin's signature.
 func (s *PluginScanner) validateSignature(plugin *PluginBase) *PluginError {
-	// For the time being, we choose to only require back-end plugins to be signed
-	// NOTE: the state is calculated again when setting metadata on the object
-	if !plugin.Backend || !s.requireSigned {
-		return nil
-	}
-
-	if plugin.Signature == PluginSignatureValid {
```

Comment:
I think removing this is a bug, since the code line 392 will execute for plugins with valid signatures.

Reply:
It's not removed though, just moved. I essentially just moved the `if plugin.Root != nil {` block, but the diff comes out weird 😞